runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -172,10 +174,12 @@ public Token appendCheckpoint(Token snapshotTimestamp) {
             startCheckpoint(snapshotTimestamp);
             int entryCount = appendObjectState(entries);
             finishCheckpoint();
-            long cpDuration = System.currentTimeMillis() - start;
+            MicroMeterUtils.time(Duration.ofMillis(System.currentTimeMillis() - start), "checkpoint.timer",
```

Comment:
I see in the codebase that some metics measure `  MicroMeterUtils.time(Duration.ofMillis(...))` (e.g. transaction.duration) and others use  `MicroMeterUtils.time(Duration.ofNanos(...))` (e.g. table.scan). @PavelZaytsev do you know if this is expected?

Reply:
In the end, it will all be in microseconds. The precision is up to a caller